src/components/InboxScreen.stories.js

```diff
@@ -17,7 +22,6 @@ const store = {
   dispatch: action('dispatch'),
 };
 
-storiesOf('InboxScreen', module)
-  .addDecorator(story => <Provider store={store}>{story()}</Provider>)
-  .add('default', () => <PureInboxScreen />)
-  .add('error', () => <PureInboxScreen error="Something" />);
+export const Default = () => <PureInboxScreen />;
```

Comment:
I used the PascalCase convention which was recently introduced to the SB codebase. This resolves the issue with `_default`, although not with the fact that the story names are quite different to all the assets (images/videos) throughout the tutorial.

I think we might just have to wear that. Otherwise we might do something like:

```js
export const Default = () => <PureInboxScreen />;
Default.story = { parameters: { name: 'default' } };
```

Which would just be a weird thing to do in the tutorial (what is the reason from the user's perspective?). I'd rather the code snippets made sense in isolation at the cost of the assets being slightly wrong.

Reply:
```js
Default.story = { name: 'default' }
```

Reply:
Are you just correcting my mistake above or do you disagree with the overall point?

Reply:
Just correcting it FYI